drivers/fastboot/fb_mmc.c

```diff
@@ -486,3 +501,67 @@ void fastboot_mmc_erase(const char *cmd, char *response)
 	       blks_size * info.blksz, cmd);
 	fastboot_okay(NULL, response);
 }
+
+/**
+ * fastboot_lock_critical() - Disable writing to special
+ * partitions like eMMC bootloader See eMMC spec for more
+ * details
+ *
+ * @cmd_parameter: Pointer to command parameter
+ * @response: Pointer to fastboot response buffer
+ */
+void fastboot_lock_critical(char *cmd_parameter, char *response)
+{
+	u8 ack = 1;
+	u8 part_num = 1;
```

Comment:
Actually we have a variable in nitrogen8m.h that says which part to use for env, maybe we should have the same for U-Boot, in case someone wants to use it from another partition.
I could live with it, it's just that I'd like to avoid hardcoded values.

Reply:
adding macros and removed the hard codings